### ISSUE TITLE
fsck remove duplicate duplicates

### ISSUE THREAD

**Issue by bobobo1618** [google/skicka#88]
I know fsck is experimental but it'd be a great help to have options for it to remove files that produce this:

`at least two instances of this file are non-empty but have different contents. Leaving them alone.`

Personally, I want it to remove the older of the duplicates.

**Comment by mmp** [comment]
This seems like a reasonable enhancement. Any suggestions for naming the command line flag?

e.g. `skicka fsck --remove-duplicates-strategy=keep-newest` seems a bit wordy. :-)

**Comment by bobobo1618** [comment]
`--remove-dupes=keep-newest`?
